src/main/java/se/kth/sda/skeleton/user/UserService.java

```diff
@@ -19,7 +20,16 @@ public User findUserByEmail(String email) {
 
     public void register(User user) {
```

Comment:
Pls could you add javadocs to this method?

Reply:
will do

Reply:
although it's Kwabena's method...maybe we should ask him to add javadoc :))))

Reply:
add it to the evaluation  survey haha

Reply:
heheh...the javadocs are there now